src/name_bazaar/ui/components/share_buttons.cljs

```diff
@@ -1,8 +1,30 @@
-(ns name-bazaar.ui.components.share-buttons)
+(ns name-bazaar.ui.components.share-buttons
```

Comment:
Share buttons don't serve only to share My Offerings page, but also for offering detail page. Must do it general purpose components with passed props.